sql/create_bldgs_table.sql

```diff
@@ -89,7 +89,7 @@ select distinct on (registrations.bbl)
   firstdeeds.docamount as lastsaleamount,
   case
     when coalesce(pluto.unitsres, 0) < 11 then NULL
-    else coalesce(oca.eviction_filings_since_2017, 0) 
```

Comment:
can you explain this change?

Reply:
properties will only show up in the OCA tables if there are eviction filings, so if the value is missing we can call that zero, except in cases of buildings with < 11 units where we are suppressing data.

Reply:
oh sorry i meant why it moved from being `oca.eviction_filings_since_2017` to just `eviction_filings_since_2017` ?

Reply:
ah okay, that makes sense thank you!! that happened here, yes? https://github.com/JustFixNYC/nycdb-k8s-loader/pull/117

Reply:
yes that's right!